docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -76,6 +76,7 @@ redirect:
 | ---| ---| ---| --- |
 |``secret`` | The name of a secret with a TLS certificate and key. The secret must belong to the same namespace as the VirtualServer. The secret must be of the type ``kubernetes.io/tls`` and contain keys named ``tls.crt`` and ``tls.key`` that contain the certificate and private key as described [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls). If the secret doesn't exist or is invalid, NGINX will break any attempt to establish a TLS connection to the host of the VirtualServer. If the secret is not specified but [wildcard TLS secret](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments#cmdoption-wildcard-tls-secret) is configured, NGINX will use the wildcard secret for TLS termination. | ``string`` | No |
 |``redirect`` | The redirect configuration of the TLS for a VirtualServer. | [tls.redirect](#virtualservertlsredirect) | No | ### VirtualServer.TLS.Redirect |
+|``certmanager`` | The certmanager configuration of the TLS for a VirtualServer. | [tls.certmanager](#virtualservertlscertmanager) | No | ### VirtualServer.TLS.CertManager |
```

Comment:
Should this be `cert-manager` to match the field in the schema?

Reply:
Good catch, thank you!